plugins/chat/db/migrate/20231207135641_add_user_chat_thread_memberships_on_thread_id_user_id_index.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class AddUserChatThreadMembershipsOnThreadIdUserIdIndex < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def up
+    execute <<~SQL
+    DROP INDEX CONCURRENTLY IF EXISTS idx_user_chat_thread_memberships_on_thread_id_user_id
+    SQL
+
+    execute <<~SQL
+    CREATE INDEX idx_user_chat_thread_memberships_on_thread_id_user_id
```

Comment:
```suggestion
    CREATE INDEX CONCURRENTLY idx_user_chat_thread_memberships_on_thread_id_user_id
```

Reply:
yes, must have deleted it at some point thx